Source/WebCore/page/Navigation.cpp

```diff
@@ -110,5 +133,13 @@ void Navigation::updateCurrentEntry(UpdateCurrentEntryOptions&&)
 {
 }
 
+// https://html.spec.whatwg.org/multipage/nav-history-apis.html#has-entries-and-events-disabled
+bool Navigation::hasEntriesAndEventsDisabled() const
+{
+    if (window()->securityOrigin() && window()->securityOrigin()->isOpaque())
```

Comment:
All on one line:
```
return window()->securityOrigin() && window()->securityOrigin()->isOpaque();
```

Reply:
Sure, this method will have a lot more if checks so I adopted that pattern for now, but until then I can make it one line.